<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>509</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="wireframe">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Wireframe</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="quit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Quit</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="draw">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Draw</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" rowspan="3">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>GroupBox</string>
        </property>
        <widget class="QSlider" name="raideur">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>251</width>
           <height>20</height>
          </rect>
         </property>
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>40</number>
         </property>
         <property name="value">
          <number>20</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QPushButton" name="restart">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>30</y>
           <width>131</width>
           <height>29</height>
          </rect>
         </property>
         <property name="text">
          <string>Restart Animation</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>161</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Constante de raideur :</string>
         </property>
        </widget>
        <widget class="QSlider" name="vent">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>190</y>
           <width>251</width>
           <height>21</height>
          </rect>
         </property>
         <property name="value">
          <number>2</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>160</y>
           <width>121</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Force du vent :</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>240</y>
           <width>121</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Pas de temps :</string>
         </property>
        </widget>
        <widget class="QSlider" name="temps">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>280</y>
           <width>251</width>
           <height>21</height>
          </rect>
         </property>
         <property name="maximum">
          <number>130</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>320</y>
           <width>131</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Att√©nuation :</string>
         </property>
        </widget>
        <widget class="QSlider" name="attenuation">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>350</y>
           <width>251</width>
           <height>20</height>
          </rect>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="value">
          <number>200</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLabel" name="K_value">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>80</y>
           <width>63</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>20</string>
         </property>
        </widget>
        <widget class="QLabel" name="Kw_value">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>160</y>
           <width>63</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>0.02</string>
         </property>
        </widget>
        <widget class="QLabel" name="T_value">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>240</y>
           <width>63</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>0.01</string>
         </property>
        </widget>
        <widget class="QLabel" name="A_value">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>320</y>
           <width>63</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>0.02</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="layout_scene">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>902</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
